### DISCUSSION TITLE
Overview of all containers created by AIO

### DISCUSSION THREAD

**Discussion by staticdev** [nextcloud/all-in-one#1570] [Questions] [answered]
Is there some documentation where I see the created containers from AIO and what is the responsibility (processed/services) of each one?

**Comment by szaimen** [comment]
Hi, why would you need such an overview/documentation?

**Reply by staticdev** [reply]
I am evaluating solutions for self-hosting basically. I have a proprietary NAS and thr vendor has some undesired limitations for creating containers.

Also I am maintainer for an Ansible role to install Nextcloud, but it does not use containers, which in turn has its own disadvantages.

Such documentation is also really useful for developers to be able to contributie to the project.

**Comment by szaimen** [comment, marked as answer]
I still don't completely understand what you want to do and why you need this documentation but okay.

Unfortunately we don't have such a documentation available. 

However, here are the build files for all containers https://github.com/nextcloud/all-in-one/tree/main/Containers. You can look at each Dockerfile what is installed and how it is started.

For a better overview of the most important containers, you might look at this file if you know how to read docker-compose files: https://github.com/nextcloud/all-in-one/blob/main/manual-install/latest.yml

If you need help understanding things, feel free to ask! :)

**Reply by staticdev** [reply]
Will take a look, make I can make a PR later to add it.
